about.md

```diff
@@ -4,18 +4,53 @@ title: About
 is_nav: true
 permalink: /about/
 ---
+開発、改善、海賊
 
+Astral Dynamics is an Innovation and Rapid Proto-typing agency. We are a multi-
+disciplinary team that brings genuine creativity together with exception 
+technical knowledge and skill. We respond to challenges, design solutions and rapid prototype.
 
-Astral Dynamics is a forward thinking tech startup with a focus on improving and enhancing culture.
+We follow our own Agile methodology and have a unique organisational culture based 
```

Comment:
Every design & innovation company ever does most of the above things. How many successful ones can you find that put it on their website? We need to tell people what we're good at. Not the way in which we actually do it.